SWLOR.Game.Server/Feature/EquipmentStats.cs

```diff
@@ -94,7 +94,7 @@ public static void RemoveStats()
         /// <param name="isAdding">If true, we're adding the HP, if false we're removing it</param>
         private static void ApplyHPBonus(uint player, uint item, ItemProperty ip, bool isAdding)
         {
-            var amount = GetItemPropertyCostTableValue(ip);
+            var amount = GetItemPropertyCostTableValue(ip) * 5;
```

Comment:
I agree, but I would like to do that closer to release when we've tested these changes more thoroughly.  If the correct multiplier is 3 or 10 or something I would rather not change all items twice.

Also - I think the easier way to do this is to update the display value in the iprp file?  Then people will see whatever we put there rather than needing to update the items themselves.